.github/workflows/docs.yml

```diff
@@ -42,7 +42,9 @@ jobs:
       - uses: actions/checkout@v3
       - name: Setup Python
         uses: actions/setup-python@v4
+      - name: Set up venv
+        run: python -m venv .pyenv
       - name: Install tox
-        run: python -m pip install tox
-      - name: Run
-        run: tox -e docs
+        run: .pyenv/bin/python -m pip install tox --break-system-packages
+      - name: Run tox
```

Comment:
I think rather than using `--break-system-packages`, I'd prefer to either pass `--user` (if that works), or install `pipx` if necessary and then use that to install tox (although it probably still requires `--user` to install `pipx`, so that doesn't really gain anything), or use the virtualenv from the previous step to install tox; or maybe it's time to switch to [the official tox GitHub Action](https://github.com/tox-dev/tox-gh). The setup for the official action seems somewhat more complicated, but it might be worth it in the long run.